templates/partials/layout/nav.phtml

```diff
@@ -26,6 +26,7 @@
         <ul class="dropdown-menu">
             <li><a href="/addEvent">New Event</a></li>
             <li><a href="/events-past">Past Events</a></li>
+            <li><a href="/calendar">Calendar</a></li>
```

Comment:
can we have a separater between this link to differentiate it from the event pages (see the applicants drop down). maybe also put it first in the list?

Reply:
isn't that there already?